docs/zk-stack/components/zkEVM/bootloader.md

```diff
@@ -49,8 +49,7 @@ We also ensure that transactions do not exceed the limits of the memory space al
 
 ## Transaction Types and Validation
 
-While the main transaction format is the internal []`Transaction`
-format](<https://github.com/code-423n4/2023-10-zksync/blob/ef99273a8fdb19f5912ca38ba46d6bd02071363d/code/system-contracts/contracts/libraries/TransactionHelper.sol#L25>),
+While the main transaction format is the internal [`Transaction` format](https://github.com/code-423n4/2023-10-zksync/blob/ef99273a8fdb19f5912ca38ba46d6bd02071363d/code/system-contracts/contracts/libraries/TransactionHelper.sol#L25),
```

Comment:
Nice! I think linking from the ever-public [era-contracts](https://github.com/matter-labs/era-contracts/blob/4aa7006153ad571643342dff22c16eaf4a70fdc1/system-contracts/contracts/libraries/TransactionHelper.sol#L25) repo is best.

Reply:
yep, fixed